src/tox_travis.py

```diff
@@ -24,6 +28,14 @@ def tox_addoption(parser):
     matched = match_envs(declared_envs, desired_envs)
     os.environ.setdefault('TOXENV', ','.join(matched))
 
+    # Travis virtualenv do not provide `pypy3`, which tox tries to execute.
+    # This doesnt affect Travis python version `pypy3`, as the pyenv pypy3
+    # is in the PATH.
+    # https://github.com/travis-ci/travis-ci/issues/6304
+    # Force use of the virtualenv `python`.
+    if version and default_factors and version.startswith('pypy3.3-5.2-'):
+        default_factors['pypy3'] = 'python'
+
```

Comment:
Thanks for including the issue in the comment here, so that we know what needs to happen for us to remove this workaround.